### DISCUSSION TITLE
Japan losing channels

### DISCUSSION THREAD

**Discussion by Lolhase12** [iptv-org/community#1145] [Q&A]
Why has Japan been losing channels?

**Comment by roxasmason274** [comment]
I noticed that half of it's channels are gone.

**Comment by Rafitart0907** [comment]
I just noticed that. Those were my favorite channels. =(

**Reply by Rafitart0907** [reply]
Nevermind. They're back again now.

**Reply by roxasmason274** [reply]
its gone again

**Comment by juney2008** [comment]
While I'm might not be an expert when it comes to IPTV stuff or channel related stuff, I most likely think that it would be the cause of the stations of the removed Japanese channels in question finding out about their channels being streamed by a server without authorization which resulting them to take down the servers in question.

There could be other reasons about this too but that's what I mostly think when it comes with a IPTV file having the server taken down.

**Comment by chugor** [comment]
Yes, Why? 😭

**Reply by jf980000** [reply]
But What About Adding Dual Audio To Japanese Channels like WOWOW Prime Plus and Cinema 4K Star Channel STAR BS10

**Comment by letterfist** [comment]
[github.com/orgs/iptv-org/discussions/1951](url)

**Reply by Solitaire242** [reply]
I turned on TV Tokyo On Air... and suddenly TBS television came on. If you could please fix TV Tokyo On Air again, I would be grateful.

**Comment by chugor** [comment]
Is it fixed?

**Comment by Solitaire242** [comment]
Yes, it's fixed.

2024년 11월 4일 (월) 오후 5:34, chugor ***@***.***>님이 작성:

**Reply by chugor** [reply]
but I still cannot stream JP channel, is there some setting I need to do?

**Comment by Solitaire242** [comment]
Try asking the admin to add other Japanese channels.

2024년 11월 4일 (월) 오후 5:38, chugor ***@***.***>님이 작성:

**Reply by chugor** [reply]
so we can only waiting............

**Comment by Alstruit** [comment]
Some channels can be found here:
https://github.com/luongz/iptv-jp

**Comment by MhysTMC4** [comment]
NHK World Japan isn't working. Could please someone fix it?

**Reply by freearhey** [reply]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-report-a-broken-stream
